Formula/feroxbuster.rb

```diff
@@ -0,0 +1,19 @@
+# typed: false
+# frozen_string_literal: true
+
+# :-)
+class Feroxbuster < Formula
+  desc "Fast, simple, recursive content discovery tool written in Rust. 🦀"
+  homepage "https://github.com/epi052/feroxbuster"
+  url "https://github.com/epi052/feroxbuster/releases/download/v1.1.0/x86_64-macos-feroxbuster.tar.gz"
```

Comment:
It needs to build from source, binaries are not allowed as CI tells you as well.

Reply:
Okay makes sense sorry again for that, I thought this would be a ci bug because homebrew on linux doesn't fail because of that.